llvm.sh

```diff
@@ -94,13 +88,54 @@ case ${DISTRO} in
         ;;
 esac
 
+while getopts ":hm:n:" arg; do
+    case $arg in
+    h)
+        usage
+        ;;
+    m)
+        BASE_URL=${OPTARG}
+        ;;
+    n)
+        CODENAME=${OPTARG}
+        LINKNAME=-${CODENAME}
```

Comment:
This is fine on the surface, however if the user passed in "testing", eg. distros like Kali (although not a practical daily driver example) where you'll want to mirror testing or unstable, you'll want to let the script flow to get to the "debian" case to set the `CODENAME` as "unstable", so that "testing" will be a valid input as well

Reply:
Yes, thought about this. Is it reasonable to enumerate some names like testing and rewrite them to unstable when the user input fail to do wget just for another try?

Reply:
Sorry, took sometime to learn how to interact between forks, here is a commit of everything we've discussed:
https://github.com/jonnysoe/llvm-jenkins.debian.net/commit/a53dad1f3a49cb8295a1d4d5c1e48422b8c78836

Reply:
I wasn't too sure what was the intent of `-n`, idk if you wanted to try to allow ubuntu to use LLVM/unstable, so remove these 2 lines if its not the case:
https://github.com/jonnysoe/llvm-jenkins.debian.net/commit/a53dad1f3a49cb8295a1d4d5c1e48422b8c78836#diff-4daaef2f715761e8c3db79addaa267b1b7284adb67def9f32acac219beb25cd9R112-R113